html/button.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html>
+   <head>
+      <style>
+      div {
+        border-radius: 25px;
+        padding: 10px;
+        height: 40px;
+        width: 100px;
+        background-color: #63d297; 
+        color: #ffffff; 
+        text-align: center; ;
+      }
+      </style>
+   </head>	
+   <body>
+       <div>Summit!</div>
```

Comment:
- please use `<button>` element for a button
- please include a screenshot of the button in the PR's description

Reply:
![image](https://github.com/hemt1912/sydney/assets/137360459/c52961aa-5397-4b63-a7dc-0534b4a8f231)

Reply:
Done!